test/integration/scan_test.go

```diff
@@ -1177,3 +1219,13 @@ func TestScanWithPolicyTimeout(t *testing.T) {
 	err, _ := executeCommand(t, args...)
 	assert.Error(t, err, "--policy-timeout should be equal or higher than 0")
 }
+
```

Comment:
tests are written twice, need to check tests for permission too, and not exits project. let me know when finish then I'll review again :)